src/Veldrid/TextureViewDescription.cs

```diff
@@ -102,14 +121,54 @@ public TextureViewDescription(Texture target, uint baseMipLevel, uint mipLevels,
         /// <param name="mipLevels">The number of mip levels visible in the view.</param>
         /// <param name="baseArrayLayer">The base array layer visible in the view.</param>
         /// <param name="arrayLayers">The number of array layers visible in the view.</param>
-        public TextureViewDescription(Texture target, PixelFormat format, uint baseMipLevel, uint mipLevels, uint baseArrayLayer, uint arrayLayers)
+        /// <param name="viewType">The view type to create or <see cref="TextureViewType.Undefined"/> to detect from texture.</param>
+        public TextureViewDescription(Texture target, PixelFormat format, uint baseMipLevel, uint mipLevels, uint baseArrayLayer, uint arrayLayers, TextureViewType viewType = TextureViewType.Undefined)
         {
             Target = target;
+            if (viewType == TextureViewType.Undefined)
```

Comment:
Please surrround with braces even if it's a one-line block

Reply:
Done